### PR TITLE
Make grid responsive

### PR DESCRIPTION
**Pull request by danrot**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | ---
| Related issues/PRs | ---
| License | MIT
| Documentation PR | ---

#### What's in this PR?

This PR makes the grid responsive.

#### Why?

Because on a very narrow view port it should be displayed as a single column.

#### To Do

- [x] Fix username next to email in permission tab of contact
- [x] Fix Password Confirmation field with screen width < 520px

### CODE REVIEW

**Comment by danrot** [comment]
I really would prefer to have some visual regression testing for the grid before we merge that :-/

**Comment by chirimoya** [comment]
@danrot so far we really don't have that many forms which we need to keep an eye on. I'm +1 for merging this soonish and not really afraid of, but I'm not sure about the max-with. Maybe we should use 1140px as max-with. At least for contact & accounts forms. For the mobile version the image size is also too small. Let's discuss on monday.

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/tests/__snapshots__/FieldBlocks.test.js.snap [hunk 1]
```diff
@@ -10,15 +10,15 @@ exports[`Render block with schema 1`] = `
           <div class=\\"icons\\"><span class=\\"su-trash-alt clickable\\" aria-label=\\"su-trash-alt\\" role=\\"button\\" tabindex=\\"0\\"></span><span class=\\"su-times clickable\\" aria-label=\\"su-times\\" role=\\"button\\" tabindex=\\"0\\"></span></div>
         </header>
         <article>
-          <div class=\\"grid\\">
-            <div style=\\"width: 100%; margin-left: 0%; margin-right: 0%;\\" class=\\"item gridItem\\">
+          <div class=\\"grid grid\\">
```
Comment (chirimoya) [inline src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/tests/__snapshots__/FieldBlocks.test.js.snap]: why we have 2 times the same class?

**Comment by chirimoya** [comment]
and the tests are failing ;)

File: src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/tests/__snapshots__/FieldBlocks.test.js.snap (hunk 1 above)
Comment (danrot) [reply src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/tests/__snapshots__/FieldBlocks.test.js.snap]: It is not really the same class, one is from the `Grid` component and one from the `FieldBlocks` component.

**Comment by danrot** [comment]
I've actually just found a visual regression in one of the delivered standard templates in sulu-minimal :confused:

**Comment by chirimoya** [comment]
min: 420px
break-point: 520px
max-with: 1140px

**Pull request merged**